README.md

```diff
@@ -87,18 +87,18 @@ This gem will directly track the semantic versioning of the Twitter Bootstrap pr
 
 ## Contributing
 
-This gem is fully tested from Rails 3.1 to 4.1. We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails-boostrap) in both Ruby 1.9, 2.0, 2.1, and jRuby 1.9 mode. If you detect a problem, open up a github issue or fork the repo and help out. After you fork or clone the repository, the following commands will get you up and running on the test suite.
+This gem is fully tested from Rails 4.0 to 5.2. We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails-boostrap) in both Ruby 1.9, 2.0  and jRuby 9000. If you detect a problem, open up a github issue or fork the repo and help out. After you fork or clone the repository, the following commands will get you up and running on the test suite.
 
 ```shell
-$ bundle install
+$ bundle
 $ bundle exec appraisal install
-$ bundle exec appraisal rake test
+$ bundle exec appraisal test
```

Comment:
`bundle exec appraisal test` raises an error: `Could not find command "test".`

This should be `bundle exec appraisal rake`. That's consistent with the instructions in `CONTRIBUTING.md`.